docs/contributing/playbook.md

```diff
@@ -69,6 +69,7 @@ Here's a comparison of our two recommendations:
 |                         **Ease of Installation**                          |          :material-check: Installation through [Windows features](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v), no external downloads           | :material-alert-circle-outline: It's [officially free now](https://blogs.vmware.com/workstation/2024/05/vmware-workstation-pro-now-available-free-for-personal-use.html), but you need to tediously sign-up for a Broadcom account |
 |                             **Compatibility**                             |                                                     :material-alert-circle-outline: Enables 'Secure system,' which causes issues for some apps                                                     |                                                                       :material-check-all: No incompatibilities with apps as it's not directly in the kernel                                                                       |
 |                              **Guest Tools**                              | :material-alert-circle-outline: Annoying to work with as it's [just an RDP session](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/enhanced-session-mode), low FPS |                                                                        :material-check-all: Has a vGPU, 60fps viewport, good drag-and-drop support, stable                                                                         |
+|                           **Operating Systems**                           |                                                                       :material-check: Hyper-V is only avaliable on Windows                                                                        |                                                                                        :material-check-all: Avaliable on Linux and Windows                                                                                         |
```

Comment:
VMware available on Windows, Linux and MacOS

Reply:
This is about VMWare Workstation, not Fusion. Fusion is mentioned just after the table

Reply:
yea i just realised, sorry